recipes/tomviz/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+python:
+  - "3.7.* *_cpython"
```

Comment:
Would be a problem to build for more Python versions?

Reply:
Yes, the build times out.